### DISCUSSION TITLE
typescript rewrite when?

### DISCUSSION THREAD

**Discussion by hyd3rs** [spongedsc/SpongeChat#80] [General]
typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? typescript rewrite when? 

just fucking with y'all, if we ever did integrate typescript it would ONLY be for core and some of the backend libraries that everything else relies on (i.e. AI SDKs and the callsystems/integrations standard libraries)

**Comment by artifishvr** [comment]
linus torvalds, kill them
